instance/config.py

```diff
@@ -3,7 +3,7 @@ class BaseConfig:
     class are shared across all environments  """
     DEBUG = False
     TESTING = False
-    JWT_SECRET_KEY = 'qwertyuiopasdfghjklsdfghjklzxcvbnmdfgh'
+    SECRET = 'qwertyuiopasdfghjklsdfghjklzxcvbnmdfgh'
```

Comment:
use an environment variable instead of hard-coding the token

Reply:
okay thanks